Add Pregnancy Analysis.ipynb

carrils opened the pull request:
Putting up the notebook i've been using to work on the pregnancy probabilities @eedrummer, please review/merge thank you

carrils committed:
Add Pregnancy Analysis.ipynb

eedrummer commented:
@carrils The notebook contains hard coded paths to your home directory. Can you make the CSV file reading relative to files that exist in this repo?

carrils committed:
Relative Paths

Add in relative file path and folder with csv files from synthea, clean up comments

carrils commented:
Issue should be fixed now, i've uploaded a folder with csv files used by the notebook and it uses those instead. 
Manually replacing those files upon generating a new population within synthea will become necessary though.
